source/module_hamilt_lcao/hamilt_lcaodft/spar_dh.h

```diff
@@ -1,15 +1,17 @@
-#ifndef SPARSE_FORMAT_DH_H
-#define SPARSE_FORMAT_DH_H
+#ifndef W_ABACUS_DEVELOP_ABACUS_DEVELOP_SOURCE_MODULE_HAMILT_LCAO_HAMILT_LCAODFT_SPAR_DH_H
```

Comment:
@caic99 and @haozhihan,
why does clang-format make change in this way?

Reply:
This action has been stopped. You can change back the original name in the next PRs.